.eas/workflows/pre-release-check.yaml

```
@@ -0,0 +1,22 @@
+  build:
+    type: build
+    params:
+      platform: ios
+      profile: preview
+  e2e:
+    type: maestro
+    needs: [build]
+    environment: preview 
+    params:
+      build_id: ${{ needs.build.outputs.build_id }}
+      flow_path: | 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)